### ISSUE TITLE
[New Pipeline]: Audio-Journey: Visual+LLM-aided Audio Encodec Diffusion

### ISSUE THREAD

**Issue by lijuncheng16** [huggingface/diffusers#3826]
### Model/Pipeline/Scheduler description

 We efficiently trained an Audio Diffusion model with the aid of Alpaca augmented audio captions using AudioSet labels;
[website](https://audiojourney.github.io/)
[preprint](https://github.com/audiojourney/audiojourney.github.io/blob/main/neurIPS_2023_v1.2.pdf)
[Appendix](https://github.com/audiojourney/audiojourney.github.io/blob/main/neurIPS_2023_appendix_v1.3.pdf)
[Implementation](https://github.com/jacksonmichaels/diffusers_with_dataloader)
Weights will be released soon!

### Open source status

- [X] The model implementation is available
- [ ] The model weights are available (Only relevant if addition is not a scheduler).

### Provide useful links for the implementation
@jacksonmichaels

_No response_

**Comment by lijuncheng16** [comment]
@sanchit-gandhi
We are interested in making a pull request and get this integrated, what are the checks we need to do perform before we make a pull request?
Baseline: we will make it a separated audio_journey_pipeline.py making sure it works and will not touch existing modules in diffuser.

**Comment by sanchit-gandhi** [comment]
Hey @lijuncheng16 - just to understand better, Audio-Journey is an LDM trained on audio-text data, where the text captions are generated as pseudo-labels from an LLM?

The eval results look super promising (beats AudioLDM by quite some margin) - congrats! Are the samples on the [website](https://audiojourney.github.io) those obtained from the most performant T5 variant of the model? They seem to be super noisy compared to the equivalent [AudioLDM samples](https://audioldm.github.io).

The first thing to check regarding an integration is the **interest** from the community around this pipeline - AudioLDM ended up not being super popular (see [downloads](https://huggingface.co/models?search=cvssp), which are only a fraction of those for say Stable Diffusion), so we'd want to have some really strong evidence that the community are interested in using this model (e.g. through a super high visibility announcement post). Unfortunately we can't add every new model to the library, since this adds a huge overhead for maintenance. Thus, we've got to focus on the ones deemed to be super in-demand from the community!

If Audio-Journey doesn't fit this category, then a [community pipeline](https://github.com/huggingface/diffusers/issues/841) may be an excellent way of integrating the pipeline with `diffusers` without adding code to the main branch. LMK what you think!

**Comment by lijuncheng16** [comment]
@sanchit-gandhi Yes, that's the direction, we believe more diverse textual representation with better attention control brings better quality and eventually would be better for the audio research community.
Thank you for the comments! Our samples are from earlier iterations, and we are continuing on perfecting this model, will refresh the demo with much better quality and much better control. 
And yes, we understand the community aspect, and we will keep improving/innovating this pipeline.
We will keep you posted :)
Kudos to you guys for providing super valuable resources for diffusion!
